src/Collections/DeltaConfigCollection.php

```diff
@@ -145,8 +146,12 @@ public function isDeltaReset($class = null)
             );
     }
 
+    /**
+     * @deprecated 1.6.0 Use __unserialize() instead
+     */
     public function unserialize($serialized)
     {
+        Deprecation::notice('1.6.0', 'Use __unserialize() instead');
```

Comment:
Do we need to implement `__unserialize()` in this class explicitly to ensure the `$this->postInit()` part is done?

Reply:
It's defined on the parent class - `MemoryConfigCollection::__unserialize()`

Reply:
Right, but that implementation doesn't call `$this->postInit()` is my point.

Reply:
Ah right

Have added `__unserialize()` to this class which calls `parent::__unseralize();` and `$this->postInit();`